docs/grammar/SolidityParser.g4

```diff
@@ -169,23 +170,14 @@ locals[
 	(
 		{!$visibilitySet}? visibility {$visibilitySet = true;}
 		| {!$mutabilitySet}? stateMutability {$mutabilitySet = true;}
-		| modifierInvocation
+		| {!$free}? modifierInvocation
+		| {$free}? {!$suffixSet}? Suffix {$suffixSet = true;}
```

Comment:
so here it means: if its a free function, where basically the suffix was not yet parsed, but a `Suffix` was parsable after that function, then we just store that we parsed a suffix, right?